docs/source/getting-started/model-registry/index.rst

```diff
@@ -0,0 +1,25 @@
+Tutorial Overview
+=================
+
+The Model Registry is a centralized model store, set of APIs, and UI, to collaboratively manage the 
+full lifecycle of an MLflow Model. It provides model lineage (which MLflow experiment and run 
+produced the model), model versioning, model aliasing, model tagging, and annotations.
+
+In this tutorial, you will get up and running with the MLflow model registry in the least amount of
+steps possible. The topics in this tutorial cover:
+
+* Registering a model programmatically to the Model Registry.
+* Viewing the model in the MLflow UI.
+* Referencing a logged model.
+* Loading a logged model for inference.
+* Aliasing model versions.
+
+To get started with the tutorial, click NEXT below.
```

Comment:
Instead of referencing the "NEXT" button (which is sort of a pain to navigate to on some pages), let's create a nav button:

``` rst
.. raw:: html

    <a href="step1-register-model.html" class="download-btn">View the tutorial</a>

```